tests/csm/rest/test_csm_load.py

```diff
@@ -409,6 +415,15 @@ def test_44790(self):
         assert resp.status_code == HTTPStatus.OK, "List IAM user failed."
         user_data = resp.json()
         self.log.info("Step 1: List user response : %s", user_data)
+        existing_user = len(user_data['users'])
+        self.log.info("Existing iam users count: %s", existing_user)
+        self.log.info("Max iam users : %s", rest_const.MAX_IAM_USERS)
+        new_iam_users = rest_const.MAX_IAM_USERS - existing_user
+        self.log.info("New users to create: %s", new_iam_users)
+
+        self.log.info("Step 2: Create users in parallel")
+        result = self.csm_obj.create_multi_iam_user_loaded(new_iam_users, existing_user)
+        assert result, "Unable to create users"
```

Comment:
Can u confirm if looking at log we will be able to find which IAM user has failed?